.buildkite/commands/run-ui-tests.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash -eu
 
-TEST_NAME=$1
-DEVICE=$2
-IOS_VERSION=$3
+DEVICE=$1
+# Default to the latest iOS, with the option of overriding
+IOS_VERSION=${2:-15.0}
```

Comment:
It's not related to the PR, but it seems like specifying the iOS version in the context of execution on CI VM does not really matter, because the VM comes (from what I have seen) only with a default set of iOSes typical for the used Xcode version. Locally, it's possible to download the Simulators with the other iOS versions, but I don't think this is the realistic case for CI. For the case of Xcode 13.4.1, it comes with iOS 15.5 (this can be seen in the [log here](https://buildkite.com/automattic/wordpress-ios/builds/9686#0182ce8b-69f5-4b39-b608-84916b8a1c9e/783)). 

So, in reality, the tests were executed on 15.5, but it's not very obvious. Maybe it's worth removing the iOS version mention at all, to avoid possible confusion?

Reply:
Good point, thank you for bringing this up.

Reply:
Screenshot showing CI only has access to iOS 15.5 Simulators:

<img width="1126" alt="Untitled2" src="https://user-images.githubusercontent.com/1218433/186592282-ae88a602-c903-4d1d-8e4e-e07e81cf5311.png">


Screenshot showing that, even though `scan` receives the deployment target parameter as 15.0, the `xctestrun` it uses is from a 15.5 build. I wouldn't be surprised if `scan`/`xcodebuild` ignored that 15.0 when using an `xctestrun`.

<img width="1126" alt="Untitled3" src="https://user-images.githubusercontent.com/1218433/186592014-fced9d6b-72e1-4e9b-8aec-03a6805388b3.png">

Reply:
@pachlava, implemented your suggestion in [wordpress-mobile/WordPress-iOS@`1143b31` (#19193)](https://github.com/wordpress-mobile/WordPress-iOS/pull/19193/commits/1143b31c0394b854ec95da87276a5ce266e539d1)